### ISSUE TITLE
Link operators to official MongoDB documentation

### ISSUE THREAD

**Issue by IshanaRaina** [mattdavis0351/mongodb-labs#10]
## Problem:
The exercises in these labs contain lots of operators and methods. Linking them to MongoDB documentation will give students an extra reference to look at if they are confused.

## Proposed Solution:
Link

**Documents to be worked on:**

- [ ] [Basic Mongo.md](../blob/master/exercises/mongo-shell/basic-mongo-queries.md)
- [ ] [Intermediate Mongo.md](../blob/master/exercises/mongo-shell/intermediate-mongo-queries.md)

**Steps to prevent recurrence:**
None planned, we are going with MVP to get 🚢 ready and will purposely revisit docs to implement this iteration after v1.

See #7 for discussion surrounding this topic. There is also an example solution located in the conversation taking place on that PR.

**Comment by mattdavis0351** [comment]
I don't think you previewed this before posting it.  Your links are all messed up about what ties to which document.  

Be careful copy/pasting things, markdown is very sensitive and doesn't always copy well

**Comment by IshanaRaina** [comment]
> I don't think you previewed this before posting it. Your links are all messed up about what ties to which document.
> 
> Be careful copy/pasting things, markdown is very sensitive and doesn't always copy well

Took care of it
